.travis.yml

```diff
@@ -21,22 +21,28 @@ matrix:
       env: SYMFONY_VERSION=4.0.*
     - php: 7.2
       env: SYMFONY_VERSION=3.4.*
+    - php: 7.2
+      env:
+        - SYMFONY_VERSION=4.0.*
+        - FLYSYSTEM_VERSION=1.0.*
+        - SIMPLE_PHPUNIT_PARAMETERS=" --stop-on-skipped"
     - php: 7.2
       env: SYMFONY_VERSION=dev-master
     - php: nightly
       env: SYMFONY_VERSION=4.0.*
-  allow_failures: 
+  allow_failures:
     - env: SYMFONY_VERSION=dev-master
     - php: nightly
-    
+
 before_install:
   - bash ./.travis/exec-before.bash
 
 install:
   - travis_retry composer require "symfony/symfony:${SYMFONY_VERSION}" $COMPOSER_FLAGS
```

Comment:
while we are looking at these things, we could use `dunglas/symfony-lock:^2` resp 3 and 4 to do this. the current code will hide missing dependencies on symfony components, as it installs all of symfony. the dunglas package conflicts with everything not in the right version instead, so the CI will show if there is a symfony component being used but not declared in require/require-dev.